### DISCUSSION TITLE
maxima crashing on my windows 11 laptop

### DISCUSSION THREAD

**Discussion by Danielle1016** [wxMaxima-developers/wxmaxima#1636] [General]
My maxima crashes 3 minutes after I opened it. My maxima version is wxMaxima 21.05.2. How can I fix this?

**Comment by daute** [comment]
Hi,
this might be the following bug: https://github.com/wxMaxima-developers/wxmaxima/issues/1607
It was already fixed in release 21.11.0.

You can either install the (wxMaxima-only) installer for wxMaxima 21.11.0:
https://github.com/wxMaxima-developers/wxmaxima/releases/tag/Version-21.11.0
(you will need a Maxima installation too! Just use the updated wxMaxima, not the included one).

Or just wait some time, probably (as discussed on the Maxima mailing-list) a new Maxima release (with an updated wxMaxima of course) will be created in the near future.

Best regards, Wolfgang

**Reply by Danielle1016** [reply]
Thank you for this, it's okay now!
